datashader/reductions.py

```diff
@@ -507,6 +531,10 @@ def __init__(self, cat_column, reduction=count()):
             self.categorizer = category_codes(cat_column)
         else:
             raise TypeError("first argument must be a column name or a CategoryPreprocess instance")
+
+        if isinstance(reduction, where):
+            raise TypeError("'by' reduction cannot use a 'where' reduction")
```

Comment:
Can you make "use" more specific? Cannot accept, does not support, etc.? It's confusing to read about reductions using reductions.

Reply:
Done.